spec/system/task_spec.rb

```diff
@@ -1,36 +1,37 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
`association`を設定することで`project_id`の指定が不要になるので、確認してみてください！
```suggestion
  let(:task) { create(:task) }
```

Reply:
e8fd623